File: proyecto1/puertaEmbarque.cpp
#include "puertaEmbarque.h"

PuertaEmbarque::PuertaEmbarque(int nombre) : identificacion(nombre)
{
    disponibilidad = true;
}

void PuertaEmbarque::anunciarEmbarque(const string &puerta)
{
    cout << "Anuncio de embarque en " << identificacion << " - Puerta " << puerta << endl;
}

File: proyecto2/puertaEmbarque.py
""" from vuelos import Vuelos """


class PuertaEmbarque:
    def __init__(self, nombre):
        self.ident = nombre
        self.disponibilidad = True
        self.historial_vuelos = []

    def anunciarEmbarque(self, puerta):
        print(
            f"Anuncio de embarque en {self.ident} - Puerta {puerta}")
        
    def to_dictPE(self):
        data = {
            "nombre": "Puerta #" + str(self.ident),
        }
        if self.disponibilidad:
            data["disponibilidad"] = "Disponible"
        else:
            data["disponibilidad"] = "No disponible"
        return data
